gimme

```diff
@@ -691,6 +696,16 @@ _last_mod_timestamp() {
 	esac
 }
 
+_get_module() {
+	if [[ ! -f "${GIMME_MODULE_PREFIX}go.mod" ]]; then
+		die 'not a module'
+	fi
+	if ! grep '^go' "${GIMME_MODULE_PREFIX}go.mod" >/dev/null; then
+		die 'module has no go directive'
+	fi
+	_resolve_version "$(grep '^go' "${GIMME_MODULE_PREFIX}go.mod" | awk '{ print $2 }').x"
```

Comment:
I think we can make this a little more defensive with a bit more `awk` (and in a separate expression so `set -e` can catch any issues):

```suggestion
	local version
	version="$(awk '$1 == "go" { print $2 ".x"; exit }' "${GIMME_MODULE_PREFIX}go.mod")"
	if [ -z "$version" ]; then
		die 'module has no go directive'
	fi
	_resolve_version "$version"
```

Reply:
Ah, thanks, I like it. I was trying to find a way to do this in one step. Your awk foo is better than mine. :-)